packages/mirta-rollup/README.md

```
@@ -3,26 +3,177 @@
+    prefix: 'APP_',
   },
 })
 
+// For packages
+export default definePackageConfig({
+  cwd: process.cwd(),
+  input: 'src/main.ts',
+  external: ['some-legacy-dep'],
+})
+
 ```
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)